server/handler/experiments.go

```diff
@@ -0,0 +1,31 @@
+package handler
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/src-d/code-annotation/server/middleware"
+	"github.com/src-d/code-annotation/server/repository"
+	"github.com/src-d/code-annotation/server/serializer"
+
+	"github.com/go-chi/chi"
+)
+
+func GetExperimentDetails(w http.ResponseWriter, r *http.Request) {
+	//TODO: !ok
+	postponed, _ := w.(middleware.PostponedResponse)
+
+	experimentID, err := strconv.Atoi(chi.URLParam(r, "experimentId"))
+	if err != nil {
+		postponed.AddError(&serializer.Error{500, "Wrong format in experiment ID sent", err.Error()})
```

Comment:
`http.StatusInternalError`

Reply:
Yes, I planned to review all errors codes and texts to use constants and make them more clear.
Should I do it before merging? Just say so :dancer:

Reply:
it's a very simple change. Why postpone?

Reply:
I didn't want 500 for everything because there could be any other with more sense. But yes, that enhanced can be postponed too.
Adding `http` constants now